src/C3 DFS BFS/Q15 JS2 특정 거리의 도시 찾기.js

```diff
@@ -0,0 +1,48 @@
+function solution(city, road, distance, start, linkArray) {
+  let result = [];
+  let distances = new Array(city + 1).fill(-1);
```

Comment:
```suggestion
  let distances = Array(city + 1).fill(-1);
```

Array는 new 키워드를 생략하여도 동일하게 동작합니다 :smile: 

한 자라도 타이핑 하는 시간을 아껴야겠죠? ㅎㅎ

Reply:
지금까지 무조건 new 를 사용해야하는 줄 알았어요 꿀팁 감사합니다!